package.json

```diff
@@ -60,18 +59,13 @@
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.2.5",
-    "@babel/runtime": "^7.2.0",
-    "classnames": "^2.2.6",
     "cornerstone-core": "^2.2.8",
-    "cornerstone-math": "^0.1.8",
```

Comment:
I think this is okay. I think we mostly/only use this in the ohif-cornerstone-extension.